types/granim/granim-tests.ts

```diff
@@ -0,0 +1,185 @@
+import {Granim} from "granim";
```

Comment:
For some reason I cannot post a suggestion here,
Pick a content from this gist:
https://gist.github.com/peterblazejewicz/f0414196c7e1eddd5580769daf8a5af3